pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -126,64 +140,52 @@ func CreateOrLoadClusterInfo(context *clusterd.Context, namespace string, ownerR
 	return clusterInfo, maxMonID, monMapping, nil
 }
 
-// ValidateAndLoadExternalClusterSecrets returns the secret value of the client health checker key
-func ValidateAndLoadExternalClusterSecrets(context *clusterd.Context, namespace string) (cephconfig.ExternalCred, error) {
-	var externalCred cephconfig.ExternalCred
-
-	secret, err := context.Clientset.CoreV1().Secrets(namespace).Get(OperatorCreds, metav1.GetOptions{})
```

Comment:
For backward compatibility, we need to check for the existence of that Secret and re-use whatever we find in `clusterInfo.CephCred.Username` and `clusterInfo.CephCred.Secret`.